mlflow/temporary_db_migrations_for_pre_1_users/alembic.ini

```diff
@@ -0,0 +1,74 @@
+# A generic, single database configuration.
```

Comment:
This whole directory is a copy of the existing `alembic` directory, except containing a different set of migrations (the migration for pre-1.0 users) under the `versions` subdirectory